### PR TITLE
New Drop - Multicloud connector for AWS

### PR DESCRIPTION
**Pull request by mrhoads**
This new drop shows how to deploy the multicloud connector enabled by Azure Arc, onboard EC2 instances to Azure Arc, and view and inventory of resources from AWS in Azure.

### CODE REVIEW

File: drops/arc_multicloud_connector.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "Title": "Using the Multicloud Connector Enabled by Azure Arc ",
```
Comment (fcabrera23) [inline drops/arc_multicloud_connector.json:2]: extra space

File: drops/arc_multicloud_connector.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  "Type": "tutorial_guide",
+  "Difficulty": "Medium",
+  "ProgrammingLanguage": [
+    "CloudFormation"
+  ],
+  "Products": [
+    "Arc",
+    "Arc-enabled Servers",
+    "EC2",
+    "AWS",
```
Comment (fcabrera23) [inline drops/arc_multicloud_connector.json:21]: As a product, i woudl put AWS EC2 and then you can put AWS as a tag

**Review by fcabrera23 — CHANGES_REQUESTED** [review]
@mrhoads  minor changes required

File: drops/arc_multicloud_connector.json (hunk 1 above)
Comment (mrhoads) [reply drops/arc_multicloud_connector.json]: fixed

File: drops/arc_multicloud_connector.json (hunk 2 above)
Comment (mrhoads) [reply drops/arc_multicloud_connector.json:21]: fixed

**Pull request merged**